python/packages/zarathustra/agents/oracle_verifier/aea-config.yaml

```diff
@@ -139,3 +154,13 @@ config:
   host: 0.0.0.0
   port: 5556
   target_skill_id: zarathustra/oracle_verification:0.1.0
+---
+public_id: valory/ledger:0.19.0
+type: connection
+config:
+  ledger_apis:
+    ethereum:
+      address: ${CHAIN_ADDRESS:str}
+      chain_id: ${CHAIN_ID:int}
+      poa_chain: ${bool:false}
+      default_gas_price_strategy: ${str:eip1559}
```

Comment:
this override is required. However, we may have multiple chains we want to query latest block number for. We have done such a thing in a previous project (multi-chain) but I have not implemented that here yet